.eslintrc.js

```diff
@@ -1,64 +1,36 @@
+const path = require('path')
+
+const ERROR = 'error'
+const WARN = 'warn'
+const OFF = 'off'
+
 module.exports = {
-  "env": {
-    "browser": true,
-    "es6": true,
-    "jest": true
+  env: {
+    browser: true,
+    jest: true,
   },
-  "globals": {
-    "__DEV__": true
+  extends: ['@instacart/eslint-config'],
+  rules: {
+    'no-plusplus': OFF,
```

Comment:
Is this just to allow for loops? If so, you can use the `allowForLoopAfterthoughts` option.